CHANGELOG.md

```diff
@@ -35,14 +35,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 </details>
 
-<details><summary><b>Removed</b></summary>
-
-</details>
-
 <details><summary><b>Fixed</b></summary>
 
 - Fixed an issue where palette index 255 was incorrectly showing as black.
 
+- Fixed instances of `CameraMan:GetScrollTarget()` and `CameraMan:SetScrollTarget()` supplying a player index instead of a screen index.
```

Comment:
Please try to write the symptom that this is fixing, not the cause without context - this is a bit meaningless to anyone playing the game